boxoffice/models/discount_policy.py

```diff
@@ -43,6 +45,9 @@ class DiscountPolicy(BaseScopedNameMixin, db.Model):
     percentage = db.Column(db.Integer, nullable=True)
     # price-based discount
     is_price_based = db.Column(db.Boolean, default=False, nullable=False)
+    discount_code_base = db.Column(db.Unicode(20), nullable=True)
```

Comment:
That's a bug. A fresh installation will not run through migrations. It'll just create schema from the code and stamp it with the last migration number. You need to remove the line from the init migration and do a manual cleanup of production.